docker-compose.override.yml

```diff
@@ -1,10 +1,40 @@
 # Development configuration
 # docker-compose up will automatically find and apply this configuration
 
-version: "3.1"
+version: "3.2"
 services:
   db:
     ports:
     - "3306:3306"
     environment:
       MYSQL_ALLOW_EMPTY_PASSWORD: "yes"
+
+  prometheus:
```

Comment:
these components are now excluded from prod? why?

Reply:
`docker stack` does not pay any attention to the build instructions, so I've moved them explicitly to the dev config.

Reply:
got it, thanks!